src/dodal/devices/apple2_undulator.py

```diff
@@ -130,50 +211,56 @@ def __init__(self, prefix: str, name: str = ""):
                 Name of the Id device
 
         """
-
-        # Gap demand set point and readback
-        self.user_setpoint = epics_signal_rw(
-            str, prefix + "GAPSET.B", prefix + "BLGSET"
-        )
-        # Nothing move until this is set to 1 and it will return to 0 when done
         self.set_move = epics_signal_rw(int, prefix + "BLGSETP")
+        # Nothing move until this is set to 1 and it will return to 0 when done.
+        super().__init__(self.set_move, prefix, name)
 
-        # These are gap velocity limit.
         self.max_velocity = epics_signal_r(float, prefix + "BLGSETVEL.HOPR")
         self.min_velocity = epics_signal_r(float, prefix + "BLGSETVEL.LOPR")
-        # These are gap limit.
-        self.high_limit_travel = epics_signal_r(float, prefix + "BLGAPMTR.HLM")
-        self.low_limit_travel = epics_signal_r(float, prefix + "BLGAPMTR.LLM")
-
-        # This is calculated acceleration from speed
-        self.acceleration_time = epics_signal_r(float, prefix + "IDGSETACC")
-
-        with self.add_children_as_readables(StandardReadableFormat.CONFIG_SIGNAL):
-            # Unit
-            self.motor_egu = epics_signal_r(str, prefix + "BLGAPMTR.EGU")
-            # Gap velocity
-            self.velocity = epics_signal_rw(float, prefix + "BLGSETVEL")
-        with self.add_children_as_readables(StandardReadableFormat.HINTED_SIGNAL):
-            # Gap readback value
-            self.user_readback = epics_signal_r(float, prefix + "CURRGAPD")
-        super().__init__(self.set_move, prefix, name)
+        """ Clear the motor config_signal as we need new PV for velocity."""
+        self._describe_config_funcs = ()
+        self._read_config_funcs = ()
+        self.velocity = epics_signal_rw(float, prefix + "BLGSETVEL")
```

Comment:
Is it possible to get this signal aliased to `prefix + ".VELO"` by the ID group? It would mean that this implementation of Motor only extends and adds signals, rather than modifying the `config_signals`, which if the base `Motor` implementation changes will miss out on those other changes.

Reply:
But the current signal is always returning 0 you said- is it viable to replace that signal with the actual calculation by the ID group?